### PR TITLE
Update README and change to JuliaSmoothOptimizers

### PR DESCRIPTION
**Pull request by abelsiqueira**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,40 +1,79 @@
...
 This is work in progress to create an interface to CUTEst based on
-[NLP.jl](https://github.com/JuliaOpt/MathProgBase.jl/tree/master/src/NLP).
+[NLPModels](https://github.com/JuliaSmoothOptimizers/NLPModels.jl).
 
 ## Installing
 
-To install CUTEst, we suggest using the [Homebrew](https://github.com/Homebrew)
-[CUTEst tap](https://github.com/dpo/homebrew-cutest), which will build a
-shared library.
+Currently, this package builds it's own version of CUTEst. If you have a working
```
Comment (dpo) [inline README.md]: its, not it's.

File: README.md [hunk 2]
```diff
@@ -1,40 +1,79 @@
...
+if nlp.meta.ncon > 0
+  f, c = cfn(nlp, nlp.meta.x0)
+  f, c = cfn(nlp.meta.nvar, nlp.meta.ncon, nlp.meta.x0)
+else
+  f = ufn(nlp, nlp.meta.x0)
+  f = ufn(nlp.meta.nvar, nlp.meta.x0)
+end
+````
+There is also a wrapper for the Fortran functions directly, which you probably
+doesn't need.
```
Comment (dpo) [inline README.md]: don't

**Comment by abelsiqueira** [comment]
Sorry about the bad english. Fixed.

**Comment by pkofod** [comment]
What about documenting the `grad!`-syntax? I use that for the Optim benchmarks at least.

**Comment by dpo** [comment]
We'll add that to the list.

**Pull request merged**